core/domain/exp_jobs_one_off.py

```diff
@@ -489,41 +488,46 @@ def map(item):
             return
 
         exploration = exp_services.get_exploration_from_model(item)
+        # err_dict is a dictionary to store the invalid tags and the
+        # invalid parent-child relations.
```

Comment:
Minor nit: add: 'that we find' to the end of this comment (or something along those lines).

Reply:
Done!